gstools/krige/tools.py

```diff
@@ -44,17 +41,19 @@ def no_trend(*args, **kwargs):
     return 0.0
 
 
-def eval_func(func, pos, mesh_type="structured"):
+def eval_func(func, pos, dim, mesh_type="structured"):
```

Comment:
Would it make sense to put the argument `dim` to the first position to make it consistent with e.g. `CovModel`?

Reply:
I can do so, but this is a low-level function users shouldn't use. But following [Hyrum's Law](https://github.com/dwmkerr/hacker-laws#hyrums-law-the-law-of-implicit-interfaces) it could be a good idea to be consistent.

Reply:
Thinking about it twice, I think we should keep it this way. It is low-level and the first parameter should hold the function to evaluate, as the function name states.